scripts/microbe_stage/microbe_operations.as

```diff
@@ -895,29 +895,31 @@ bool validPlacement(CellStageWorld@ world, ObjectID microbeEntity, const Organel
     return touching;
 }
 
-// speciesName decides the template to use, while individualName is
-// used for referencing the instance
 ObjectID spawnMicrobe(CellStageWorld@ world, Float3 pos, const string &in speciesName,
-    bool aiControlled)
+    bool aiControlled, bool partOfColony)
 {
     assert(world !is null);
     assert(speciesName != "");
 
     if(pos.Y != 0)
         LOG_WARNING("spawnMicrobe: spawning at y-coordinate: " + pos.Y);
 
-
     // Create microbeEntity with correct template, physics and species name
-    auto microbeEntity = _createMicrobeEntity(world, aiControlled, speciesName,
-        // in_editor
-        false);
+    auto microbeEntity = _createMicrobeEntity(world, aiControlled, speciesName, false);
 
     // Teleport the cell to the right position
     auto microbePos = world.GetComponent_Position(microbeEntity);
     microbePos._Position = pos;
     microbePos.Marked = true;
 
     auto physics = world.GetComponent_Physics(microbeEntity);
+    
+    auto speciesEntity = findSpeciesEntityByName(world, speciesName);
+    auto species = world.GetComponent_SpeciesComponent(speciesEntity);
+
+    if (species.isBacteria)
+    { physics.Body.SetMass(physics.Body.Mass * 10); }
```

Comment:
This formatting isn't what the C++ code would have. Also why does the bacteria get the mass multiplied by a ton?

Reply:
I actually don't know why the bacteria's mass is multiplied by that much. This was the code in the previous spawnBacteria function, I just copied it over.

Reply:
That might be the change @Untrustedlife made to make the bacteria harder to push around. But anyway the formatting needs to be changed here.